bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -21,434 +20,288 @@ class Neuron(CMakePackage):
     """
 
     homepage = "https://www.neuron.yale.edu/"
-    url = "http://www.neuron.yale.edu/ftp/neuron/versions/v7.5/nrn-7.5.tar.gz"
+    url = "https://github.com/neuronsimulator/nrn/releases/download/8.2.3/full-src-package-8.2.3.tar.gz"
```

Comment:
This is great! I download this and then wildly guess later which software it actually is when I look at `~/Downloads`?

Reply:
I thought the same when I copied the URL 🙄 Agree, we need to fix.